blog post about delta 1.1 and 1.2 (#318)
deltachat/deltachat-pages: _posts/2020-03-06-delta-1.2.md

---
title: "Pin Chats, Provider Database, Offline Help and more"
author: "bjørn, paula petersen (drawing)"
image: ../assets/blog/2020-03-06-drawing.png
---

<img src="../assets/blog/2020-03-06-drawing.png" width="270" style="float:left; margin-right:1em;" /> 
In the first months this year, we concentrated very much on fixing bugs,
adding features and getting releases out - and a bit less on blogging and marketing.

Might be, this is not optimal, however, i have the feeling it is better than the other way round :)

Anyway, here we are:

### What is going on feature-wise in Delta Chat 1.1 and 1.2?

<img src="../assets/blog/2020-03-06-pin-parents.png" width="270" style="float:right; margin-left:1em; margin-bottom:1em;" />
**Never forget what is _really_ important:** Delta Chat now allows to "pin" any chat,
so it is always visible atop of the chat list.
In combination with the "archive chats" function, you have some nice tools to keep your things in order.

**_Lots_ of traffic-saving improvements,** too many for this blogpost.

> We will get into details in a separate post therefore.

Reviewer: ```suggestion
We will get into details with a separate blog post soon.
```

:woman_shrugging: not very important, just feels better :)

Author: i better leave it as it is. these "soon" promises are considered dangerous :)

**Instant QR-code contact set up** - Delta Chat uses QR-codes for easy contact setup -
no need to type in complicated e-mail addresses or to compare even more complicated fingerprints.
- _In the past_, this took a moment as both devices had to be online and you had to wait for the handshake to finish.  
- _Now_ the handshake runs in background and there is no need to wait for it.
You can now start typing your first message even if one or both devices is offline!

> **Built-in help and FAQ** just in case you want some help on some feature, it is now just a fingertip away.

Reviewer: ```suggestion
**Built-in help and FAQ** just in case you need some help, it is now just a fingertip away.
```

nitpicking^^ not important, feel free to ignore, just feels better to my ears.

<img src="../assets/blog/2020-03-06-provider-db.png" width="270" style="float:left; clear:both; margin-right:1em;" />
**Know what to do when logging in to your e-mail provider:** As you probably know, 

> Delta Chat does not has own servers, messages are sent with the provider of your choice.

Reviewer: ```suggestion
Delta Chat does not have own servers, messages are sent with the provider of your choice.
```

grammar :)

For whatever reasons, some providers will require you to do some stuff before your can really use them.  
This is not the fault of Delta Chat - 
but if we know about these things, we will now show some hints on the log in screen.

As most things in Delta Chat, also the provider-database and the built-in-help are **integrated in the app**,

> do not load information from external servers and cannot track you or can cause troubles on bad networks.

Reviewer: ```suggestion
As most things in Delta Chat, also the provider-database and the built-in-help are **integrated in the app**.
They do not load information from external servers, they do not track you, they can not cause troubles on bad networks.
```
Nitpicking, shorter sentences just make it a bit easier to read.
